### DISCUSSION TITLE
Display angular outputs as actions by default

### DISCUSSION THREAD

**Discussion by Jakob-em** [storybookjs/storybook#14285] [Ideas]
As the official angular coding style guide mentions not to prefix outputs with `on` ([link](https://angular.io/guide/styleguide#dont-prefix-output-properties)) the default method for detecting actions (`argTypesRegex`) is not working out of the box for those outputs.
I was wondering if the information from compodoc about the components (which includes a list of outputs) could be used to automatically create the actions for those Outputs.

Is this something that should be handled in storybook itself or via an addon, what do you think?

**Comment by shilman** [comment]
I think this should get handled in the compdooc processor: https://github.com/storybookjs/storybook/blob/next/addons/docs/src/frameworks/angular/compodoc.ts

Any interest in adding it?

**Reply by Jakob-em** [reply]
Sure, I will have a look at it.
Just wanted to make sure that I am on the right track.
